types/react/index.d.ts

```diff
@@ -2591,6 +2591,9 @@ declare namespace React {
     //   - "string"
     //   - union of string literals
     interface SVGAttributes<T> extends AriaAttributes, DOMAttributes<T> {
+        // React-specific Attributes
+        suppressHydrationWarning?: boolean | undefined;
```

Comment:
Can we just move these to `DOMAttributes`?

Reply:
So far `DOMAttributes` only contain callbacks + `children` and `dangerouslySetInnerHTML` properties. Some other common ones are duplicated between `SVGAttributes` and `HTMLAttributes`, e.g. `className`, `id`, `name` etc. So I don't know about that, maybe it's a subject of a bigger refactoring? But it's also my first PR here so I don't really know what would be better 😄